### PR TITLE
Find any state or city news

### PR DESCRIPTION
**Pull request by Kunal614**

### CODE REVIEW

File: web_programming/google_news.py [hunk 1]
```diff
@@ -0,0 +1,110 @@
+
+
+
+import requests
```
Comment (Kunal614) [inline web_programming/google_news.py]: Request module

File: web_programming/google_news.py [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+from bs4 import BeautifulSoup
+from fake_useragent import UserAgent
+import webbrowser
+import emoji
+
+import time
+def Times_of_India(url,ua,url1):
```
Comment (Kunal614) [inline web_programming/google_news.py]: Data extracted from times of India

File: web_programming/google_news.py [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+        if bol=='y':
+            #print(data1.get('href'))
+            url1+=data1.get('href')
+            print("%s"%url1)
+
+            webbrowser.open(url1)
+    return len(data)        
+          
+
+def news_india(url, ua , url1):
```
Comment (Kunal614) [inline web_programming/google_news.py]: Data Extracted from News India

File: web_programming/google_news.py [hunk 4]
```diff
@@ -0,0 +1,110 @@
...
+            if bol=='y' or bol == 'Y':
+                data2 = data[i].find('a')
+                url1=data2.get('href')
+                #print("%s"%url1)
+                webbrowser.open(url1)
+            
+        
+        
+    return len(data)    
+
```
Comment (Kunal614) [inline web_programming/google_news.py]: Main function

**Comment by Kunal614** [comment]
Request for changes

**Comment by cclauss** [comment]
File conflicts to resolve, No type hints, Automated tests are failing.

**Comment by cclauss** [comment]
Spellcheck...
./web_programming/google_news.py:31: avaliable  ==> available
./web_programming/google_news.py:62: avaliable  ==> available

File: google_news.py [hunk 5]
```diff
@@ -0,0 +1,110 @@
+
```
Comment (cclauss) [inline google_news.py]: Please remove this file.  We do not allow Python files at the root directory
.

File: web_programming/crawl_google_results.py [hunk 6]
```diff
@@ -5,6 +5,21 @@
 from fake_useragent import UserAgent
 import requests
 
+<<<<<<< HEAD
```
Comment (cclauss) [inline web_programming/crawl_google_results.py]: Incomplete git merge.

File: web_programming/google_news.py [hunk 7]
```diff
@@ -0,0 +1,139 @@
...
+"""
+
+import requests
+from bs4 import BeautifulSoup
+import webbrowser
+import time
+
+
+def Times_of_India(userInput, ua):
```
Comment (cclauss) [inline web_programming/google_news.py]: type annotations please as discussed in CONTRIBUTING.md

File: web_programming/google_news.py [hunk 8]
```diff
@@ -0,0 +1,139 @@
...
+        "\n\033[1;35;40m Search any news (state , city ,Country , AnyThings etc) : ",
+        bold_end,
+        end=" ",
+    )
+
+    userInput = input()
+
+    print(bold_start, "\033[1;33;40m \n")
+    print("Which news channel data would you prefer")
+    print("1. Times of india")
```
Comment (cclauss) [inline web_programming/google_news.py]: Capital I in India.

**Pull request closed without merging**